src/App.vue

```diff
@@ -419,61 +462,100 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         console.log('Launch Darkly update error =', error) // eslint-disable-line no-console
       })
 
-      // ensure user is authorized to use this IA
-      await this.checkAuth().catch(error => {
-        console.log('Auth error =', error) // eslint-disable-line no-console
-        this.accountAuthorizationDialog = true
-        throw error // go to catch()
-      })
-
       try {
-        // fetch draft filing
-        let draftFiling = await this.fetchDraft()
-
-        // if there is an existing filing, check if it is in a valid state to be edited
-        if (draftFiling) {
-          this.invalidIncorporationApplicationDialog = this.hasInvalidFilingState(draftFiling)
-          if (this.invalidIncorporationApplicationDialog) return
-        }
-
-        // merge draft properties into empty filing so all properties are initialized
-        const emptyFiling = this.buildFiling()
-        draftFiling = { ...emptyFiling.filing, ...draftFiling }
-
-        // parse draft filing into the store
-        if (draftFiling) {
-          this.parseDraft(draftFiling)
+        let draftFiling, resources
```

Comment:
This code looks OK except it's pretty big. I suggest making 2 sub-methods from it (and the common stuff can stay here).

Reply:
My initial pass was as you suggest, ended up combining them. Can break them out though, no problem :)

Reply:
It will make it easier to review :)

If I was looking at the code in an IDE then I'd just fold the sub-sections for the same effect.

Reply:
Totally, i 100% agree with you. I can improve this 👍

Reply:
updated in next commit!